### ISSUE TITLE
upgrade our lnds to 0.13.1

### ISSUE THREAD

**Issue by nicolasburtey** [GaloyMoney/galoy#364]

**Comment by dolcalmi** [comment]
`checkIsBalanced()` is failing after channel opening in tests, I'm not sure if it's about fees, but if it's not urgent I'll check on the weekend

Image used in docker-compose: `lncm/lnd:v0.13.1-monitoring`

**Comment by nicolasburtey** [comment]
this is not urgent

**Comment by dolcalmi** [comment]
## Premise (according to our code)

- After open the channel: **capacity - commit_transaction_fee - local_balance = 0**


## Problem

we are using commit_transaction_fee to calculate escrow in Lnd but v0.13.1 is returning an inconsistent commit_transaction_fee or there is an additional fee that is not parsed by lightning package (altough I can't see anything new in https://api.lightning.community/#lnrpc-channel)

### lncm/lnd:v0.12.1-monitoring
**capacity - commit_transaction_fee - local_balance = 0**
`1000000 - 9050 - 990950 =` **0** (ok)

<details>
  <summary>data from updateEscrows method in lndUtils.ts</summary>

```json
{
  diff: 9050,
  escrowInLnd: 9050,
  escrowInMongodb: 0,
  channels: [
    {
      "capacity": 1000000,
      "commit_transaction_fee": 9050,
      "commit_transaction_weight": 600,
      "id": "142x1x0",
      "is_active": true,
      "is_anchor": false,
      "is_closing": false,
      "is_opening": false,
      "is_partner_initiated": false,
      "is_private": false,
      "is_static_remote_key": true,
      "is_variable_remote_key": false,
      "local_balance": 990950,
      "local_csv": 144,
      "local_dust": 573,
      "local_given": 0,
      "local_max_htlcs": 50,
      "local_max_pending_mtokens": "990000000",
      "local_min_htlc_mtokens": "1",
      "local_reserve": 10000,
      "partner_public_key": "038e4ed546cf24c19b14a59437ac53e7c89ecd2d3265bd35ab3dcea0856002944d",
      "pending_payments": [],
      "received": 0,
      "remote_balance": 0,
      "remote_csv": 144,
      "remote_dust": 573,
      "remote_given": 0,
      "remote_max_htlcs": 50,
      "remote_max_pending_mtokens": "990000000",
      "remote_min_htlc_mtokens": "1",
      "remote_reserve": 10000,
      "sent": 0,
      "time_offline": 0,
      "time_online": 1000,
      "transaction_id": "3825fbea33833db038b62494f97a7269429dfe0674a8a19648c8be32f3c58ae4",
      "transaction_vout": 0,
      "unsettled_balance": 0
    }
  ]
}
```
</details>

### lncm/lnd:v0.13.1-monitoring
**capacity - commit_transaction_fee - local_balance = 0**
`1000000 - 3140 - 996530 =` **330** (error)

<details>
  <summary>data from updateEscrows method in lndUtils.ts</summary>

```json
{
  diff: 3140,
  escrowInLnd: 3140,
  escrowInMongodb: 0,
  channels: [
    {
      "capacity": 1000000,
      "commit_transaction_fee": 3140,
      "commit_transaction_weight": 772,
      "id": "142x1x0",
      "is_active": true,
      "is_anchor": true,
      "is_closing": false,
      "is_opening": false,
      "is_partner_initiated": false,
      "is_private": false,
      "is_static_remote_key": false,
      "is_variable_remote_key": false,
      "local_balance": 996530,
      "local_csv": 144,
      "local_dust": 573,
      "local_given": 0,
      "local_max_htlcs": 50,
      "local_max_pending_mtokens": "990000000",
      "local_min_htlc_mtokens": "1",
      "local_reserve": 10000,
      "partner_public_key": "0215b957f83758b3ec76fdd50f6c25a59a5c720cd0c37ad1cacd203c883fe613a0",
      "pending_payments": [],
      "received": 0,
      "remote_balance": 0,
      "remote_csv": 144,
      "remote_dust": 573,
      "remote_given": 0,
      "remote_max_htlcs": 50,
      "remote_max_pending_mtokens": "990000000",
      "remote_min_htlc_mtokens": "1",
      "remote_reserve": 10000,
      "sent": 0,
      "time_offline": 0,
      "time_online": 1000,
      "transaction_id": "624d80132c0cc9c06fd193efd9be7aa9c57cd5c9bfc0ee5aace7368c9e2ee303",
      "transaction_vout": 0,
      "unsettled_balance": 0
    }
  ]
}
```
</details>

**Comment by nicolasburtey** [comment]
@dolcalmi escrow changed by default with lnd 0.13

look at "Default Anchor Output Channels" in https://lightning.engineering/posts/2021-06-17-lnd-v0.13/

I believe if you deactivate anchor output the tests should pass?

**Comment by krtk6160** [comment]
Closed by #511
